main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"path/filepath"
 	"strings"
 	"syscall"
-	template_text "text/template"
+	texttemplate "text/template"
```

Comment:
As below, underscore is not an anti-pattern for that use case.
I generally prefer the common ground to be the prefix rather than the suffix.